service/worker/archiver/util.go

```diff
@@ -127,3 +130,33 @@ func errorDetails(err error) string {
 	err.(*cadence.CustomError).Details(&details)
 	return details
 }
+
+func historiesEqual(fetchedHistory *clientShared.History, uploadedHistory *shared.History) (bool, string) {
```

Comment:
Return error instead of string?

Reply:
same comment as above.

Reply:
Thank you :) I ended up deleting the function all together.